session.html

```diff
@@ -76,12 +84,25 @@ <h1>Viewing Session #11223344:</h1>
     
     <!-- Search Function -->
     <script>
-    $(document).ready(function() {
-      $("#searchInput").on("keyup", function() {
-        var value = $(this).val().toLowerCase();
-        $("#tableContents tr").filter(function() {
-          $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1)
-        });
+    var $rows = $("#session_table tr:not(:first)");
```

Comment:
this search script is used twice. can we just write a generic version for it? pass in an argument for either #session_table or #database_table